custom/up_nrhm/data_sources/asha_facilitators.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
@snopoke Run load_spec command causes KeyError: 'doc_type'. Adding doc_type causes another error BadValueError: Property domain is required. Should I do something before load_spec or add some param?